crates/common/validator/src/validator.rs

```diff
@@ -0,0 +1,39 @@
+use anyhow::{Result, ensure};
```

Comment:
```suggestion
use anyhow::{ensure};
```

as @syjn99 said we import `anyhow::Result` manually